### DISCUSSION TITLE
have 2 dependent dropdown values

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#7458] [Feature Requests]
I was wondering if we can have 2 dropdowns and when we select 1 entry in the first one to show different options in the second one?

**Comment by benhaynes** [comment]
Yes, Conditional Fields can adjust other field interface options: https://docs.directus.io/guides/fields/#conditions

**Reply by botzill** [reply]
But I mean, if I select 1 option in 1 dropdown, change all the options in another? Is this possible ?

**Reply by benhaynes** [reply]
Yes, as I referenced. Those docs talk about setting rules for Field A, and Interface Options for Field B.

**Reply by botzill** [reply]
Perfect, thank you.

**Reply by haidv1992** [reply]
how create Address of city, district, ward?

**Comment by haidv1992** [comment]
https://user-images.githubusercontent.com/86423592/130899766-06abc67d-2ffc-4be2-bb75-3a821994984d.mov
